scripts/signature_collector.py

```diff
@@ -0,0 +1,11 @@
+import logging
+
```

Comment:
Don't we need `sys.path.append('.')` for this to work?

Reply:
works on my machine :smiley: 
and in docker:
```
# Set PYTHONPATH to include /app so Python knows where to find your modules
ENV PYTHONPATH=/app
```